doc/classes/EditorSettings.xml

```diff
@@ -576,6 +576,9 @@
 		<member name="interface/editor/font_antialiasing" type="int" setter="" getter="">
 			FreeType's font anti-aliasing mode used to render the editor fonts. Most fonts are not designed to look good with anti-aliasing disabled, so it's recommended to leave this enabled unless you're using a pixel art font.
 		</member>
+		<member name="interface/editor/font_clamp_glyph_offset" type="float" setter="" getter="">
+			Sets default value for glyph offset clamping. Value is a fraction of font height. If set to zero or negative value, no clamping is done.
```

Comment:
Out of curiosity, why does this need to be an editor setting (as opposed to say, a project setting)?

Reply:
I guess project setting also should be added.

Reply:
Moved to project settings.